Formula/python.rb

```diff
@@ -98,17 +98,25 @@ def install
     ldflags  = []
     cppflags = []
 
-    unless MacOS::CLT.installed?
-      # Help Python's build system (setuptools/pip) to build things on Xcode-only systems
+    if !MacOS::CLT.installed? || MacOS.version >= :mojave
+      # Help Python's build system (setuptools/pip) to build things on
+      # Xcode-only systems or systems where headers are not in /usr/include
+      if !MacOS::CLT.installed?
+        sdk = MacOS.sdk_path
+      else
+        sdk = "/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk"
```

Comment:
`MacOS.sdk_path` should return something equivalent - we probably need to adjust this logic to check for `MacOS::CLT.provides_headers?` in addition to `!MacOS::CLT.installed?`, and then we don't have to hardcode this path.